lib/meta-box-partial-page.php

```diff
@@ -359,6 +365,32 @@ function the_gutenberg_metaboxes() {
 		printf( "<script type='text/javascript'>\n%s\n</script>\n", trim( $script ) );
 	}
 
+	/**
+	 * If the 'postcustom' meta box is enabled, then we need to perform some
+	 * extra initialization on it.
+	 */
+	$enable_custom_fields = get_option( 'enable_custom_fields', false );
+	if ( $enable_custom_fields ) {
+		$script = <<<JS
+			( function( $ ) {
+				if ( $('#postcustom').length ) {
+					$( '#the-list' ).wpList( {
+						addBefore: function( s ) {
+							s.data += '&post_id=' + $('#post_ID').val();
```

Comment:
Should `$('#post_ID').val();` be cast to a number?

Reply:
I borrowed this from [Core](https://github.com/WordPress/wordpress-develop/blob/master/src/js/_enqueues/admin/post.js#L672). Looking at it some more, we could avoid the jQuery selector altogether by just interpolating `$post->ID`.

Fixed in 86b5b4e9a.